src/presentation/http/router/note.test.ts

```diff
@@ -244,7 +252,109 @@ describe('Note API', () => {
         expect(response?.json().accessRights).toStrictEqual({ canEdit: true });
       });
     });
+  });
+
+  describe('PATCH note/:notePublicId ', () => {
+    test('Update note by public id with 200 status, user is creator of the note', async () => {
+      const expectedStatus = 200;
+      const userId = 2;
+      const accessToken = global.auth(userId);
+
+      const userNote = notes.find(newNote => {
+        return newNote.creator_id === userId;
+      });
+
+      const response = await global.api?.fakeRequest({
+        method: 'PATCH',
+        headers: {
+          authorization: `Bearer ${accessToken}`,
+        },
+        url: `/note/${userNote!.public_id}`,
+        body: {
+          'content': { new: 'content added' },
+        },
+      });
+
+      expect(response?.statusCode).toBe(expectedStatus);
+    });
```

Comment:
we decided not to use expectedStatus anymore

Reply:
So we should use only numbers? Like
`expect(response?.statusCode).toBe(200);`?

Reply:
yes